requirements.txt

```diff
@@ -6,3 +6,4 @@ scikit-learn
 scipy<1.14.0
 tqdm
 psutil
+sparsekmeans
```

Comment:
Please also add sparsekmeans to `install_requires` https://github.com/ntumlgroup/LibMultiLabel/blob/a0bef91d770d449dce6cc5a1a0fb8dc1c9d0647e/setup.cfg#L27-L35

Reply:
Please bump version to 0.8.0
https://github.com/ntumlgroup/LibMultiLabel/blob/a0bef91d770d449dce6cc5a1a0fb8dc1c9d0647e/setup.cfg#L3

Reply:
@khoinpd0411 No need to bump version now, we will release with #20.